devnotes/documentation.md

```diff
@@ -118,7 +131,7 @@ In order to build coreboot image, follow the steps below:
    (assuming in the coreboot root directory):
 
    ```
-   scp build/coreboot.rom root@<IP>:/tmp
+   scp build/coreboot.rom root@<BMC_IP>:/tmp
```

Comment:
```suggestion
   scp build/coreboot.rom.signed.ecc root@<BMC_IP>:/tmp
```
```
If that file is not present, use `coreboot.rom` instead
```

(GH suggestions won't let me put three backticks, and when I try to escape them with backslash, that backslash is also printed)

This will change after SEEPROM changes are merged, so it is probably better to default to newer version.

The same has to be done below in `pflash` in step 3 but GH won't let me add a comment there.

Reply:
Changed it to `coreboot.rom.signed.ecc`